src/js/tabs/static_tab.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+TABS.staticTab = {};
+TABS.staticTab.initialize = function (staticTabName, callback) {
+    var self = this;
+
+    if (GUI.active_tab != staticTabName) {
+        GUI.active_tab = staticTabName;
+    }
+    var tabFile = "./tabs/" + staticTabName + ".html";
+    if (staticTabName == 'changelog') { tabFile = './changelog.html'; }
```

Comment:
We could probably avoid this hack if we changed where changelog is put by the gulp task.